_progress-bar.scss

```diff
@@ -0,0 +1,206 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__variable-name: true !default;
+
+// Override Font Awesome's default path
+$fa-font-path: "/bower_components/font-awesome/fonts";
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+@import 'font-awesome/scss/font-awesome';
+
+
+@mixin state-colors($color) {
+    .c-progress-bar__label {
+        color: $color;
+    }
+
+    .c-progress-bar__progress {
+        background-color: $color;
+    }
+
+    .c-progress-bar__spinner-progress {
+        stroke: $color;
+    }
+}
+
+// Progress Bar root
+// ---
+
+.c-progress-bar {
+    display: flex;
+    height: $tap-size;
+    margin: 0 15px;
+
+    align-items: center;
+}
+
+
+// Process Bar sub-component
+// ---
+
+.c-progress-bar__spinner-section {
+    &:not(:empty) {
+        position: relative;
+
+        width: 45px;
+        height: 45px;
+        margin-right: 10px;
+    }
+
+    .c-progress-bar__spinner {
+        .c-progress-bar__spinner-background {
+            stroke: $neutral-50;
+        }
+    }
+
+    .c-progress-bar__text {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+    }
+}
+
+.c-progress-bar__bar-section {
+    margin-right: 5px;
+
+    flex-grow: 1;
+
+    .c-progress-bar__background {
+        overflow: hidden;
+        height: 5px;
+
+        background: $neutral-50;
+    }
+
+    .c-progress-bar__progress {
+        transform: translate3d(-100%, 0, 0);
+
+        height: 5px;
+        width: 100%;
+
+        background: $accent-color;
+
+        transition: transform 0.5s ease-in;
+    }
+}
+
+.c-progress-bar__extra-section {
+    position: relative;
+
+    width: 35px;
+    height: 25px;
+
+    .c-progress-bar__text {
+        position: absolute;
+        right: 0;
+        bottom: -3px;
+    }
+}
+
+.c-progress-bar__indicator {
+    width: 25px;
+    height: 25px;
+
+    text-align: center;
+
+    .c-progress-bar__spinner-section & {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
```

Comment:
@cole-sanderson for future reference, try to avoid linting warnings or errors in PR comments. While important, they pull focus away from criticizing the actual code itself.